packages/browser-agent-core/src/features/spa/aggregate/index.js

```diff
@@ -216,6 +217,7 @@ export class Aggregate extends FeatureBase {
         // so that it has a chance to possibly start an interaction.
         if (INTERACTION_EVENTS.indexOf(evName) !== -1) {
           var ixn = new Interaction(evName, this[FN_START], state.lastSeenUrl, state.lastSeenRouteName, onInteractionFinished, agentIdentifier)
+          state.prevInteraction = ixn
```

Comment:
Maybe a naming confusion, but wouldn't you want to assign the previous interaction before the new Interaction? Or is this acting as the last seen?  (there's another `new Interaction` on ln 523 -- is that flow pertinent?)